ansible/roles/esdc-cfgdb/files/query_cfgdb

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+ZKCLI=/usr/local/bin/zookeepercli
+QUERY_STRING="$@"
+
+if [ -z "${QUERY_STRING}" ]; then
+                echo "Usage:   $0 <query>" 1>&2
+                echo "Example: $0 ls /esdc/nodes" 1>&2
+                exit 10
+fi
+
+if [ ! -x "${ZKCLI}" ]; then
+                echo "Error: Cannot find binary ${ZKCLI}" 1>&2
+                exit 20
+fi
+
+# query cfgdb credentials
+CFGDB_USERNAME="$(mdata-get org.erigones:cfgdb_username 2>/dev/null || echo 'esdc')"
+CFGDB_PASSWORD="$(mdata-get org.erigones:cfgdb_password 2>/dev/null)"
+
+if [[ -n "${CFGDB_USERNAME}" && -n "${CFGDB_PASSWORD}" ]]; then
+	"${ZKCLI}" -servers localhost -auth_usr "${CFGDB_USERNAME}" -auth_pwd "${CFGDB_PASSWORD}" -c ${QUERY_STRING}
+else
+	"${ZKCLI}" -servers localhost -c ${QUERY_STRING}
+fi
+
+exit $?
```

Comment:
Does this return the exit code of the `"${ZKCLI}"` command or of the `if` statement?